[2025-02-12T00:36:16.909+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T00:36:16.971+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-12T00:34:12.728305+00:00 [queued]>
[2025-02-12T00:36:16.994+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-12T00:34:12.728305+00:00 [queued]>
[2025-02-12T00:36:16.994+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2025-02-12T00:36:17.022+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): assign_permissions_dag> on 2025-02-12 00:34:12.728305+00:00
[2025-02-12T00:36:17.030+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=624) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-12T00:36:17.033+0000] {standard_task_runner.py:72} INFO - Started process 625 to run task
[2025-02-12T00:36:17.035+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'assign_permissions_dag', 'manual__2025-02-12T00:34:12.728305+00:00', '--job-id', '262', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpqrhjr840']
[2025-02-12T00:36:17.039+0000] {standard_task_runner.py:105} INFO - Job 262: Subtask assign_permissions_dag
[2025-02-12T00:36:17.135+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-02-12T00:34:12.728305+00:00 [running]> on host 3f11f16c6f4b
[2025-02-12T00:36:17.342+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='assign_permissions_dag' AIRFLOW_CTX_EXECUTION_DATE='2025-02-12T00:34:12.728305+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-12T00:34:12.728305+00:00'
[2025-02-12T00:36:17.343+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-12T00:36:20.695+0000] {dag.py:230} INFO - Assigned permissions to dados for DAG hello_world_dag
[2025-02-12T00:36:23.950+0000] {dag.py:230} INFO - Assigned permissions to data-team for DAG manage_***_roles
[2025-02-12T00:36:23.952+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T00:36:23.959+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-12T00:36:23.959+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=assign_permissions_dag, run_id=manual__2025-02-12T00:34:12.728305+00:00, execution_date=20250212T003412, start_date=20250212T003616, end_date=20250212T003623
[2025-02-12T00:36:23.990+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T00:36:24.024+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T00:36:24.026+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
